src/oic/oauth2/message.py

```diff
@@ -470,6 +406,58 @@ def _add_value(self, skey, vtyp, key, val, _deser, null_allowed):
                 else:
                     self._dict[skey] = val
 
+    def _add_value_list(self, skey, vtype, key, val, _deser, null_allowed):
+        """Add value that is supposed to be a list.
```

Comment:
Can you improve on this docstring? Not sure it gives a good overview.

Reply:
Tried to improve, not sure if it is any better:)